components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LogConstants.java

```diff
@@ -58,4 +58,96 @@ public static class InputKeys {
         public static final String ERROR_MESSAGE = "error message";
         public static final String CALLER_PATH = "common auth caller path";
     }
+
+    /**
+     * Application Mgt related log constants.
+     */
+    public static class ApplicationManagement {
+        public static final String CREATE_APPLICATION_ACTION = "create-application";
+        public static final String UPDATE_APPLICATION_ACTION = "update-application";
+        public static final String DELETE_APPLICATION_ACTION = "delete-application";
+        public static final String CREATE_SAML_APPLICATION_ACTION = "create-saml-application";
+        public static final String DELETE_SAML_APPLICATION_ACTION = "delete-saml-application";
+        public static final String UPDATE_SAML_APPLICATION_ACTION = "update-saml-application";
+        public static final String CREATE_OAUTH_APPLICATION_ACTION = "create-oauth-application";
+        public static final String UPDATE_OAUTH_APPLICATION_ACTION = "update-oauth-application";
+        public static final String DELETE_OAUTH_APPLICATION_ACTION = "delete-oauth-application";
+        public static final String REGENERATE_CLIENT_SECRET_ACTION = "regenerate-client-secret";
+        public static final String UPDATE_APP_STATE_ACTION = "update-app-state";
+    }
+
+    /**
+     * User mgt related log constants.
+     */
+    public static class UserManagement {
+        public static final String ADD_USER_ACTION = "add-user";
+        public static final String DELETE_USER_ACTION = "delete-user";
+        public static final String SET_USER_CLAIM_VALUE_ACTION = "set-user-claim-value";
+        public static final String SET_USER_CLAIM_VALUES_ACTION = "set-user-claim-values";
+        public static final String DELETE_USER_CLAIM_VALUES_ACTION = "delete-user-claim-values";
+        public static final String DELETE_USER_CLAIM_VALUE_ACTION = "delete-user-claim-value";
+        public static final String CREDENTIAL_UPDATE_BY_USER_ACTION = "credential-update-by-user";
+        public static final String CREDENTIAL_UPDATE_BY_ADMIN_ACTION = "credential-update-by-administrator";
+        public static final String GET_USER_LIST_ACTION = "get-user-list";
+        public static final String GET_USER_CLAIM_VALUE_ACTION = "get-user-claim-value";
+        public static final String GET_USER_CLAIM_VALUES_ACTION = "get-user-claim-values";
+        public static final String UPDATE_USERS_OF_GROUP = "update-users-of-group";
+        public static final String UPDATE_GROUPS_OF_USER_ACTION = "update-groups-of-user";
+
+        public static final String ADD_GROUP_ACTION = "add-group";
+        public static final String GET_GROUP_ACTION = "get-group";
+        public static final String DELETE_GROUP_ACTION = "delete-group";
+        public static final String UPDATE_GROUP_ACTION = "update-group";
+        public static final String UPDATE_GROUP_NAME_ACTION = "update-group-name";
+        public static final String GET_GROUP_LIST = "get-group-list";
```

Comment:
```suggestion
        public static final String GET_GROUP_LIST_ACTION = "get-group-list";
```
Check below constants as well